content/_guides/changing-metadata.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Customizing Platform Page Metadata
```

Comment:
This title is a bit misleading because the word `metadata` is so generic. Let's rephrase the title to `Customizing HTML meta tags`?